src/Sections/EntitiesSection/Entities/Leader.ts

```diff
@@ -0,0 +1,52 @@
+import Entity, { CommonEntityOptions } from '../Entity';
+import { BoundingBox, boundingBox_t } from 'Internals/BoundingBox';
+import { Dxfier } from 'Internals/Dxfier';
+import { vec2_t, point3d } from 'Internals/Helpers';
+
+export enum ArrowHeadFlags {
+    Disabed = 0,
+    Enabled = 1,
+}
+
+export enum PathType {
+    StraightLine = 0,
+    Spline = 1,
+}
+
+export interface LeaderVertex {
+    point: vec2_t;
+}
```

Comment:
The leader vertex its only a position so no need for a new structure, and also can have a z component so why not use an array of vec3_t:
```ts
public constructor(vertices: vec3_t[], options?: LeaderOptions)
```
and in dxfy:

```ts
for (const v of this.vertices) {
    dx.point3d(v);
}
```

Reply:
done sir